taker-electron/.gitignore

```diff
@@ -17,5 +17,6 @@ npm-debug.log.*
 
 /target
 /testnet
+/dist
```

Comment:
nit: commit message is a bit misleading, you're actually *removing* generated dirs from the project 🙃